### DISCUSSION TITLE
How to set a new seprate database?

### DISCUSSION THREAD

**Discussion by mrezdev** [cjmellor/level-up#35] [Ideas]
In this package you create many models & databases, I want to define a second database for these package
I have many errors when connecting to the second DB, like:
Relation between user in migration files & setup db connection in your models
Is it possible to define a config for your package to connect to another database?!

**Comment by cjmellor** [comment]
The package has a config, which you can publish with `php artisan vendor:publish` and choose the `level-up-config` option.

This will allow you to change any settings and add your own.

**Reply by mrezdev** [reply]
I know that & I changed the user setting in the config file but the package has some models that need to be modified
, and I think we need something extra in the config file, that supports the new database.

**Reply by cjmellor** [reply]
I don't really have any desire to support multiple databases as it's very unique. 

If you need to amend the Models, you could just create your own Models and extend the existing ones and overwrite any methods where you need to change things to suit your unique situation.
